scopes/dependencies/dependency-resolver/policy/variant-policy/variant-policy.ts

```diff
@@ -23,7 +23,7 @@ export type VariantPolicyEntryValue = {
   resolveFromEnv?: boolean;
 };
 
-export type DependencySource = 'auto' | 'env' | 'slots' | 'config';
+export type DependencySource = 'auto' | 'env' | 'env-default-peer' | 'slots' | 'config';
```

Comment:
i'm not sure we need the word default here
env-peer is enough

Reply:
maybe worth not say peer here, as maybe tomorrow the env will have more policies on itslef.
an example (which we might really want to implement) is the following:
i build a react18 env. I have react 18 in my peers.
I have @types/react 18 in the dev deps in the getDependencies.
so bit show on the env itself will show react 18 as runtime.
but the @types/react will be 17 (coming from the envs/env or the aspect env)
which of course is a bit weird. I would probably want to take to my types the types that i'm setting for my components if I have a peer with the same name.
in that case this will also be calculated here probably, and in that case the name env-peer is not correct anymore.
so maybe env-self?